app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
```

Comment:
The condition `len(command.split()) == 3` is correct, but you should store the result of `command.split()` in a variable to avoid calling `.split()` multiple times, as suggested in the checklist.